<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubWindowLocalConfig</class>
 <widget class="QWidget" name="SubWindowLocalConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#widget_3
{
	border:1px solid red
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget
{
	border:1px solid red
}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item row="0" column="1">
              <widget class="QPushButton" name="getip">
               <property name="text">
                <string>获取IP</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>ip</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_getip"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>mask</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_getmask"/>
             </item>
             <item row="1" column="4">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>gateway</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLineEdit" name="lineEdit_getgateway"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_2
{
	border:1px solid red;
   background-color:#F2F1EF;
}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item row="0" column="1">
              <widget class="QPushButton" name="setip">
               <property name="text">
                <string>设置IP</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>ip</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_setip"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>mask</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_setmask"/>
             </item>
             <item row="1" column="4">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>gateway</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLineEdit" name="lineEdit_setgateway"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget#widget_4
{
	border:1px solid red
}</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="setrtc">
               <property name="text">
                <string>设置RTC</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>RTC时间</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDateEdit" name="dateEdit_setrtc"/>
             </item>
             <item row="0" column="3">
              <widget class="QTimeEdit" name="timeEdit_set"/>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="getrtc">
               <property name="text">
                <string>获取RTC</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_10">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>RTC时间</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDateEdit" name="dateEdit_getrtc"/>
             </item>
             <item row="1" column="3">
              <widget class="QTimeEdit" name="timeEdit_get"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>其它设置</string>
      </attribute>
      <widget class="QLabel" name="label_datime">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>541</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
